File: cloudwatch-alerting-sdk/java-sdk/siren-core/src/main/java/com/stagwelltech/siren/config/SirenConfig.java
package com.stagwelltech.siren.config;

import lombok.Data;

@Data
public class SirenConfig {
    public static class AwsConfig {
        public String endpoint;
        public String region;
        public String profile;
        public String accessKey;
        public String secretKey;
        public Integer httpConnectionsCount;
    }

    public boolean enabled = true;
    public String provider;
    public AwsConfig aws;
    public Boolean async = true;
    public Integer asyncWaitTime = 500;
}

File: cloudwatch-alerting-sdk/dotnet-sdk/net-standard/src/main/siren/Configs/SirenConfig.cs
using System;
using System.Collections.Generic;
using System.Text;
using YamlDotNet;
using YamlDotNet.Serialization;

namespace StagwellTech.SirenSDK.Configs
{
    public class SirenConfig
    {
        public class AzureApplicationInsights
        {
            public string instrumentationKey;
            public string appId;
            public string apiKey;
            //public string profile;
            //public string accessKey;
            //public string secretKey;
            //public int httpConnectionsCount;
        }



        public AzureApplicationInsights applicationInsights;
        public bool enabled = true;
        public bool developerMode = false;
        //public string provider;
        //public bool async = true;
        //public int asyncWaitTime = 500;

    }
}
